console/console-init/ui/src/graphql-module/queries/project.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import gql from "graphql-tag";
+
+const FILTER_RETURN_PROJECTS = (filterObject?: any) => {
+  const { projectname, projectType } = filterObject;
```

Comment:
Please revisit the name 'filterObject'. It should give me some information about what filter is it? Is this filter based on name or some other property?

Reply:
Agreed, I will define a type for filterObject when i will implement the filter section for project list

Reply:
@drcoolsanjeev I was referring to the naming here, not the type.